src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -59,6 +61,23 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @POST
+    @Path("/ExecuteIQLQuery")
+    @Accept("application/json")
+    public async executeIQLQuery(request: any): Promise<Envelope> {
+        return this.ok(await this.consumptionServices.attributes.executeIQLQuery({ query: { queryString: request.query.queryString } }));
+    }
+
+    @POST
+    @Path("/ValidateIQLQuery")
+    @Accept("application/json")
+    public validateIQLQuery(request: any): Envelope {
+        const result = validateIQL(request.query.queryString);
```

Comment:
we should not call `validateIQL` directly from here and instead wrap this to a usecase.

our UseCases do much more stuff than just executing a function. That shoud also resolve the comment above.

Reply:
why do you resolve this conversation? I think it isn't resolved.